### PR TITLE
Dawntrail - PCT Gauge suggestions, misc. fixes, and mark supported

### PR DESCRIPTION
**Pull request by AkairyuGestalter**
## Pull request type

- [x] This is new functionality or an addition to existing functionality
- [x] This is a bugfix to existing functionality

## Pull request details

- [x] The goal of this PR is detailed below:
 - Adds suggestions to the gauge module to avoid:
   - Overcapping Palette
   - Ending the fight with 50+ Palette remaining
   - Overwriting Monochrome Tones
   - Overwriting Mog/Madeen
 - Updates the ActionTimeline:
   - Configured a useful order to show PCT's oGCDs
   - Fixed a data bug that was causing Muses/Subtractive spells/Drip to show up twice in the timeline
 - Set a defined display order for PCT's analysis
 - Miscellaneous fixes:
   - Fixed the remainder of the "Pallette" spelling errors I'd made earlier
   - Updated Gauge to only reset Palette and Paint on death. The Canvas state (including Depictions and Portraits) do not reset on death.

## Testing / Validation

- [x] I used the log(s) listed below to develop and test this bugfix: http://localhost:3000/fflogs/DhYmnP1VNQX3KJAL/2/2
 - This sample log should have an example of all of the gauge overwrite/overcap errors

## Job Maintenance
- [x] I am active on the xivanalysis Discord and part of the job maintainers group for the job(s) in this PR

### CODE REVIEW

File: src/parser/jobs/pct/modules/Gauge.tsx [hunk 1]
```diff
@@ -240,17 +250,81 @@ export class Gauge extends CoreGauge {
...
+	}
+
+	private onComplete() {
+		// Suggestion to not overcap the palette gauge
+		const lostSubtractives = Math.floor(this.paletteGauge.overCap / SUBTRACTIVE_COST)
+		this.suggestions.add(new TieredSuggestion({
+			icon: this.data.actions.SUBTRACTIVE_PALETTE.icon,
+			value: lostSubtractives,
+			content: <Trans id="pct.gauge.suggestions.lost-subtractives.content">Overcapping the Palette gauge will cause you to lose uses of <DataLink action="SUBTRACTIVE_PALETTE"/> over the course of the fight, leading to lost potency. Make sure to use your gauge before it overcaps.</Trans>,
+			why: <Trans id="pct.gauge.suggestions.lost-subtractives.why">You lose <Plural value={lostSubtractives} one="# use" other="# uses" /> of <DataLink showIcon={false} action="SUBTRACTIVE_PALETTE" /> due to overcapping the Palette gauge.</Trans>,
```
Comment (AzariahAeroborn) [inline src/parser/jobs/pct/modules/Gauge.tsx]:
```suggestion
			why: <Trans id="pct.gauge.suggestions.lost-subtractives.why">You lost <Plural value={lostSubtractives} one="# use" other="# uses" /> of <DataLink showIcon={false} action="SUBTRACTIVE_PALETTE" /> due to overcapping the Palette gauge.</Trans>,
```
_Nit:_ This should be the same past tense as other suggestions

File: src/parser/jobs/pct/modules/Gauge.tsx [hunk 2]
```diff
@@ -240,17 +250,81 @@ export class Gauge extends CoreGauge {
...
+				2: SEVERITY.MAJOR,
+			},
+		}))
+
+		// Suggestion to not end the fight with palette gauge remaining
+		const wastedSubtractives = Math.floor(this.paletteGauge.value / SUBTRACTIVE_COST)
+		this.suggestions.add(new TieredSuggestion({
+			icon: this.data.actions.SUBTRACTIVE_PALETTE.icon,
+			value: wastedSubtractives,
+			content: <Trans id="pct.gauge.suggestions.wasted-subtractives.content">Ending the fight with Palette gauge remaining means you could have used another set of <DataLink action="SUBTRACTIVE_PALETTE" /> spells for additional damage.</Trans>,
```
Comment (AzariahAeroborn) [inline src/parser/jobs/pct/modules/Gauge.tsx]: How "bursty" is palette gauge gain?  Does this get into the realm of a knowing your kill time suggestion?  Seems very non-typical to me to make a recommendation like this.

**Review by AzariahAeroborn — CHANGES_REQUESTED** [review]

File: src/parser/jobs/pct/modules/Gauge.tsx (hunk 1 above)
Comment (AkairyuGestalter) [reply src/parser/jobs/pct/modules/Gauge.tsx]: Wops. Good catch, is fixed.

File: src/parser/jobs/pct/modules/Gauge.tsx (hunk 2 above)
Comment (AkairyuGestalter) [reply src/parser/jobs/pct/modules/Gauge.tsx]: The Subtractive spells are near double the potency of their additive equivalents, and I hazily thought I remembered other places where we suggest making use of leftover gauge but I'm definitely wrong there. I'm fine with ditching it.

**Review by AzariahAeroborn — APPROVED** [review]

**Pull request merged**